source/features/delete-fork-link.tsx

```diff
@@ -4,7 +4,7 @@ import features from '../libs/features';
 import {getRepoURL} from '../libs/utils';
 
 function init() {
-	const currentBranch = select('#partial-pull-merging .merge-branch-description .commit-ref');
+	const currentBranch = select<HTMLElement>('#partial-pull-merging .merge-branch-description .commit-ref');
```

Comment:
Used the generic `HTMLElement` as I have no idea what kind of element this is supposed to be. I would guess a `<div />`, but not certain.

Reply:
I'd stick to `HTMLElement`